### DISCUSSION TITLE
Start an album or playlist from the middle of it

### DISCUSSION THREAD

**Discussion by Icelk** [music-assistant/support#3158] [Q&A] [answered]
Hi!
Thank you for making a great piece of software. I have a question; is there a way for me to start playing an album on e.g. the 4th track, and then have the WHOLE album repeat? The closest thing I've seen is right click and `Play Album from here`, but that excludes the tracks before that when the queue is repeated.

One solution I could think of is to place the earlier tracks (i.e. 1-3 in the example above) in the "Previously played" part of the queue.

**Comment by OzGav** [comment, marked as answer]
You can play the album then go to the queue and select play now on track 4. This will move the first three tracks to played and you can then turn on the repeat mode (assuming you didn’t do it before)

**Reply by Icelk** [reply]
Thank you!
